INSTRUCTIONS.md

```diff
@@ -0,0 +1 @@
+In order to validate the cluster has been launched successfully make a request to http://localhost/
```

Comment:
Consider adding more detailed instructions on what the expected response should be when making a request to `http://localhost/`. Additionally, mention any prerequisites or setup steps required before making this request, such as port forwarding or ensuring the application is running.